### DISCUSSION TITLE
Command to accept all corrections

### DISCUSSION THREAD

**Discussion by douglaslassance** [wrenger/obsidian-languagetool#11] [Ideas]
Sometimes I would prefer to accept all the corrections at once and then proofread over having to click each spelling mistake one by one.

**Comment by wrenger** [comment]
Indeed a command to apply all charges could be handy. I'll look into this.

**Reply by wrenger** [reply]
I've added this in 6c1e4506a9079c078f33366a5b7dbfddc689b65f
It will be part of the next release
